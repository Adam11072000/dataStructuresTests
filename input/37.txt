init 8140
addImage 5601
addImage 2926
setLabelScore 2926 3810 1951 299
setLabelScore 5601 5990 732 2068
mergeSuperPixels 2926 1205 0
addImage 1407
quit