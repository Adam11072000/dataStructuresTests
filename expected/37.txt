init done.
addImage: SUCCESS
addImage: SUCCESS
setLabelScore: SUCCESS
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
addImage: SUCCESS
quit done.
